lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -200,7 +200,7 @@ def start_ansible_runner
         env['FOREMAN_CALLBACK_DISABLE'] = '1' if @rex_command
         env['SMART_PROXY_ANSIBLE_ENVIRONMENT_FILE'] = Proxy::Ansible::Plugin.settings[:ansible_environment_file]
         command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
-        command << '--cmdline' << cmdline unless cmdline.nil?
+        command << "--cmdline='#{cmdline}'" unless cmdline.nil?
```

Comment:
If you provide tags, don't the inner single quotes conflict with the single quotes used on L218?

Reply:
According to what I checked, it does conflict with other single quotes if we use those. 

@Ron-Lavi I tried to test the code with your changes from here (even though they are irrelevant) https://github.com/theforeman/smart_proxy_ansible/compare/master...Ron-Lavi:smart_proxy_ansible:vault and it works for me without the changes from this PR (with them it failed). 
So I'm good with closing this PR unless I missed something.